santorini-common/src/lib.rs

```diff
@@ -6,8 +6,71 @@ pub mod position;
 
 #[cfg(test)]
 mod tests {
+    use crate::{
+        command::{BuildCommand, MovementCommand},
+        objects::{
+            player::Player,
+            state::{State, StateBuilder},
+            tower::Level,
+        },
+        phase::Phase,
+        position::{Column, Position, Row},
+    };
+
     #[test]
     fn it_works() {
-        assert_eq!(2 + 2, 4);
+        let state = StateBuilder::new()
```

Comment:
Hmm, I thought we wanted to `try_fold` instead? See https://github.com/mingyli/santorini/pull/8